frontend/src/metabase/entities/questions.js

```diff
@@ -16,11 +17,21 @@ import { canonicalCollectionId } from "metabase/collections/utils";
 
 import forms from "./questions/forms";
 
+const listQuestions = GET("/api/card");
+const listModelsForDatabase = GET("/api/database/:dbId/models");
```

Comment:
Assuming we are returning `Card` objects here, would it make sense to add parameters to `/api/card` to return not all cards but only cards:
- for a specific db
- only models

The existing code won't work if we return objects of a different shape, so maybe using the `/card` endpoint would make it more obvious.

Reply:
If we end up with 2 endpoints, please define API cals in `metabase/service.js`, like here https://github.com/metabase/metabase/blob/60cb22c438e3a2545090c9ae76cf2b744e9fb12e/frontend/src/metabase/entities/timelines.js#L22

Reply:
Good q. I originally implemented it with just `api/card`. But I then posed this question to backend eng, and the consensus is we should try to hide the fact that models are implemented as cards in the API, which means favouring separate endpoints over reusing `/api/card`. I can see the pros and cons either way, but it will likely stay as separate endpoints.